### PR TITLE
Add heuristic to allow parsing 2-page simplates

### PR DESCRIPTION
**Pull request by pjz**

### CODE REVIEW

**Comment by whit537** [comment]
Needs a test for the new SyntaxError.

File: aspen/simplates/__init__.py [hunk 1]
```diff
@@ -189,6 +189,21 @@ def parse_into_pages(self, decoded):
         pages = list(split_and_escape(decoded))
         npages = len(pages)
 
+        # DWIM: if there's under three pages, the last one *must* be
+        # explicitly marked a template page, otherwise it might be python
+        # leaving us not knowing what to do.
+        if npages < 3:
+            if not pages[-1].header:
+                type_name = self.__class__.__name__[:-len('resource')]
+                msg = "%s resources must have at least one explicit template pages; %s has none."
```
Comment (whit537) [inline aspen/simplates/__init__.py]: "pages" -> "page"

**Comment by pjz** [comment]
Should all be fixed now.

**Comment by pjz** [comment]
Ugh, I'd hoped that would help with the doc site, but it doesn't, or at least, not much.

**Pull request merged**